lms/templates/admin/instance.html.jinja2

```diff
@@ -89,11 +89,14 @@
         {{ settings_checkbox("Groups enabled", "blackboard", "groups_enabled") }}
         {{ settings_checkbox("Files enabled", "blackboard", "files_enabled") }}
     </fieldset>
-
     <fieldset class="box">
         <legend class="label has-text-centered">Content integrations</legend>
         {{ settings_checkbox("Microsoft OneDrive enabled", "microsoft_onedrive", "files_enabled", default=True) }}
+
         {{ settings_checkbox("VitalSource enabled", "vitalsource", "enabled") }}
+        {{ settings_text_field("VitalSource user ID param", "vitalsource", "user_lti_param") }}
```

Comment:
At some point VS could get its own `<fieldset>` but we probably should wait until a bit of progress is made in the VS inclusive access project. 

Maybe we'll discover that some settings are for VS filepicker while other are only for VS IA.

Reply:
I tried laying these out in columns, but it looked really ugly